src/views/routing/sidebar/body/SidebarBody.native.js

```diff
@@ -16,6 +16,8 @@ class Sidebar extends Component {
     sidebarLayout: string,
     headerLayout: string,
     footerLayout: string,
+    sortDirection: string,
```

Comment:
if the only options are 'asc' and 'desc', make them the only options ie: sortDirection: oneOf( 'asc', 'desc')